omnigibson/simulator.py

```diff
@@ -1124,51 +1276,19 @@ def world_prim(self):
             """
             return lazy.omni.isaac.core.utils.prims.get_prim_at_path(prim_path="/World")
 
-        def clear(self) -> None:
```

Comment:
so with no clear, how should the sim stage be wiped? or is this no longer supported in the new og stack?

Reply:
This is moved to omnigibson/__init__.py

Reply:
Clear does exist, it's just not on Simulator because the Simulator object should not be reused post-clear. Essentially clear deletes the existing simulator object and creates a new one.